exporter/datadogexporter/metrics_exporter.go

```diff
@@ -208,5 +211,12 @@ func (exp *metricsExporter) PushMetricsData(ctx context.Context, md pmetric.Metr
 		)
 	}
 
+	if len(sp) > 0 {
+		exp.params.Logger.Debug("exporting APM stats payloads", zap.Any("stats_payloads", sp))
+		for _, p := range sp {
+			exp.apmStatsProcessor.ProcessStats(p, "", "")
```

Comment:
We can (at least for traces coming from the OTel SDKs) use the [Telemetry SDK semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk) to get the lang and tracer version values here (the second and third arguments).

Do we want to do that? (Not a blocker for this PR if `ProcessStats` works correctly with empty strings there)

Reply:
This is actually the source of the stats, used for metrics. I added the build info (command + version) here.

Reply:
Oh, okay! Yes, they were confusing to me at least